__tests__/app.test.js

```diff
@@ -4,6 +4,7 @@ const app = require('../app');
 const db = require('../db/connection');
 const seed = require("./../db/seeds/seed");
 const testData = require("./../db/data/test-data");
+require('jest-sorted');
```

Comment:
We only need to do this, and in a different file, if we are using other jest addon packages like jest-chain. This can be removed

Reply:
Also we can remove line 1, jest will set the NODE_eNV for us without us writing it

Reply:
Apologies, I don't understand. I thought I need to require jest-sort in the file in order to have access to the matcher I am relying on when testing the order of my response object. The docs asked me to require it in and when I take it out I get TypeError saying that "toBeSortedBy is not a function". What file would i therefore require it in other than the test file?

Reply:
In terms of  NODE_eNV, Sarah mentioned in her lecture to include it in although I understand it is not a necessity so I am a little confused about this too (I assumed Sarah meant it is good to include it for clarity purpose and I assumed it is good practice). I will take it out though as maybe I misunderstood!